StandardConfig/production/Calibration/Calibration_ILD_l5_o1_v09.xml

```diff
@@ -0,0 +1 @@
+Calibration_ILD_l5_o1_v02.xml
```

Comment:
The symlinks to the XMLs can be removed. For now we use these models only with Gaudi.